keyboards/tweetydabird/lotus58/blok/halconf.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 Markus Knutsson (@TweetyDaBird)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include_next <halconf.h>
+#define HAL_USE_I2C TRUE
```

Comment:
The whole file should be removed — `HAL_USE_I2C` is already enabled for all converters.

(But note that the order of `#define` and `#include_next` is wrong here - for `halconf.h` `#define` should go first, for `mcuconf.h` you need to do `#include_next` first, and then `#undef` and `#define` to override some values.)